app.js

```
@@ -201,7 +201,8 @@ app.get("/events/:event", (req,res)=>{
 });
 
 app.post("/projects/student", (req,res)=>{
-    const {studentName} = req.body;
+    let { studentName } = req.body;
```

You could also add in a check and throw a 400 if the studentName is invalid or not provided? Sorry, should've asked for this in my previous review

---

Done